source/include/jobs.h

```diff
@@ -308,7 +489,45 @@ JobsStatus_t Jobs_StartNext( char * buffer,
  * @note A jobId consisting of the string, "$next", is supported to generate
  * a topic string to request the next pending job.
  *
- * @note The thingName and jobId parameters do not need a NULL terminator.
+ * @note The thingName and jobId parameters do not need a NUL terminator.
+ *
+ * <b>Example</b>
+ * @code{c}
+ *
+ * // The following example shows usage of the Jobs_Describe API to
+ * // generate topic string for the DescribeJobExecution API
+ * // of the AWS IoT Jobs service.
+ *
+ * // Every device should have a unique thing name registered with AWS IoT Core.
+ * // This example uses a dummy serial number for the thing name.
+ * #define THING_NAME           "11223445566"
+ * #define THING_NAME_LENGTH    ( sizeof( THING_NAME ) - 1U )
+ *
+ * // The job ID is required to send a status update for a job to the AWS IoT
+ * // Jobs service.
+ * #define JOB_ID               "My_Job"
+ * #define JOB_ID_LENGTH        ( sizeof( JOB_ID ) - 1U )
+ *
+ * // This example allocated buffer of maximum length for a Jobs topic
+ * // possible for the thing name using the JOBS_API_MAX_LENGTH macro.
+ * char topicBuffer[ JOBS_API_MAX_LENGTH( THING_NAME_LENGTH ) ] = { 0 };
+ * uint16_t topicLength = 0;
+ * JobsStatus_t status = JobsSuccess;
+ * status = Jobs_Describe( topicBuffer,
+ *                         sizeof( topicBuffer ),
+ *                         THING_NAME,
+ *                         THING_NAME_LENGTH,
+ *                         pJobId,
+ *                         jobIdLength,
```

Comment:
Should be using `JOB_ID` and `JOB_ID_LENGTH`?

Reply:
Yes, fixed